torchao/prototype/mx_formats/nvfp4_tensor.py

```diff
@@ -634,6 +634,25 @@ def nvfp4_view_op(func, types, args, kwargs):
     )
 
 
+@implements([aten.select.int])
+def nvfp4_select(func, types, args, kwargs):
+    old, dim, index = args
+    assert dim == 0, f"NVFP4Tensor aten.select.int with {dim=} is not yet supported"
+    assert len(old.qdata.shape) == len(old._scale_e4m3.shape), "unsupported"
+    new = old.__class__(
+        old.qdata[index],
+        old._scale_e4m3[index],
+        old._block_size,
```

Comment:
should a dim get knocked off block size after you select?

Reply:
currently `block_size` is an integer for this tensor, 16 for NVFP4.  If we change it to a multidimensional block, we'd have to update this code.